barretenberg/cpp/pil/avm/avm_kernel.pil

```diff
@@ -25,30 +25,37 @@ namespace avm_kernel(256);
     pol commit side_effect_counter;
 
     // FIXED INDEXES
+    // Exists checks
     pol START_NOTE_HASH_EXISTS_WRITE_OFFSET = 0;
-    pol START_EMIT_NOTE_HASH_WRITE_OFFSET = 4;
-    pol START_NULLIFIER_EXISTS_OFFSET = 8;
-    pol START_EMIT_NULLIFIER_WRITE_OFFSET = 12;
-    pol START_L1_TO_L2_MSG_EXISTS_WRITE_OFFSET = 16;
-    pol START_EMIT_UNENCRYPTED_LOG_WRITE_OFFSET = 20;
-    pol START_EMIT_L2_TO_l1_MSG = 24;
+    pol START_NULLIFIER_EXISTS_OFFSET = 32; // START_NOTE_HASH_EXISTS_WRITE_OFFSET + MAX_NOTE_HASH_READ_REQUESTS_PER_CALL TODO: exists and non exists
```

Comment:
Adjusted these values to line up with the true offsets when using the real PER_CALL limits, previously they were just small ranges as an MVP

Reply:
Can you now autogen these from TS?